web-common/src/features/help/initPylonChat.ts

```diff
@@ -1,11 +1,15 @@
-import type { V1User } from "../../client";
-
 const appId = import.meta.env.RILL_UI_PUBLIC_PYLON_APP_ID as string;
 
+export type UserLike = {
+  email: string;
+  displayName: string;
+  photoUrl: string;
+};
+
```

Comment:
Nice solution to the import problem. Mind just adding a comment to explain the situation?